swift/ql/src/queries/Security/CWE-312/CleartextLogging.ql

```diff
@@ -18,7 +18,9 @@ import codeql.swift.dataflow.DataFlow
 import codeql.swift.security.CleartextLoggingQuery
 import CleartextLoggingFlow::PathGraph
 
-from CleartextLoggingFlow::PathNode src, CleartextLoggingFlow::PathNode sink
-where CleartextLoggingFlow::flowPath(src, sink)
-select sink.getNode(), src, sink, "This $@ is written to a log file.", src.getNode(),
-  "potentially sensitive information"
+from CleartextLoggingFlow::PathNode source, CleartextLoggingFlow::PathNode sink
+where CleartextLoggingFlow::flowPath(source, sink)
+select sink, source, sink,
```

Comment:
We normally use the `Node` (and not the `PathNode`) as the alert element (see https://codeql.github.com/docs/writing-codeql-queries/creating-path-queries/#constructing-a-path-query).
```suggestion
select sink.getNode(), source, sink,
```